tests/lib/network_helper.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from netaddr import *
+import re
+import ipaddress
+def mac_to_snmp_format(mac_addr):
+    if mac_addr != None:
+        try:
+            mac_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",mac_addr)
+            mac = EUI(mac_tmp,dialect=mac_unix)
+            mac_final = str(mac).upper()
+            return mac_final
+        except:
+            assert False , 'Mac Type is Error!'
+    else:
+        assert False , 'Error, MAC is None!'
```

Comment:
Did you look at my reworked version?

https://review.gerrithub.io/c/lgirdk/boardfarm/+/448492/3/tests/lib/network_helper.py

you don't need to be cathcing all these, if the parse fails this function should raise an exception we should not be catching it and just raising a new exception with our own messages...

Reply:
Okay, I just use function but not assert, right?

Reply:
Yes, please review the link I sent...